scaleway/resource_scaleway_volume_attachment.go

```diff
@@ -10,6 +10,36 @@ import (
 	"github.com/scaleway/scaleway-cli/pkg/api"
 )
 
+func init() {
```

Comment:
`init` and `testSweepVolume` need to be in `resource_scaleway_volume_attachment_test.go` file